lessons/misc-quick-reference/cs-string.md

```diff
@@ -49,50 +49,189 @@ var message = $"{name} is a {description}";
 var mathProblem = $"What do you get when you multiply six by 9: {6 * 9}"
 ```
 
----
-
 ### String length
 
----
+We can get the length of a string by using the length property.
+
+```C#
+var name = "Suncoast Developers Guild";
+
+var nameLength = name.Length;
+
+Console.WriteLine(nameLength);
+//Output: 25;
+```
 
 ### Iterating the characters in a string
 
----
+We can iterate over a string and assess each character with a foreach loop.
+
+```C#
+var name = "Mary";
+
+foreach (var character in name)
+{
+  Console.WriteLine(character);
+}
+
+//Output: 
+//M
+//a
+//r
+//y
+```
+
+Alternatively, we can use the string length to use a for loop and access each character by its index.
+
+```C#
+var name = "Mary";
+
+for (var i = 0; i < name.Length; i++)
+{
+  Console.WriteLine(name[i]);
+}
+
+//Output:
+//M
+//a
+//r
+//y
+```
 
 ### Splitting a string
 
----
+We can split a string into an array of strings using the Split method. The first parameter will be the character you want to split the string on.
```

Comment:
Realistically your example is splitting on a string, given you wrote `(" ")` -- double check your usage and if it is on a string, change this. If it should be a character, change the code below.

Reply:
Yep, you are correct. Split works on a single character from my experience. I have corrected the example to have single ticks.